docs/source/tutorials/day_of_the_week.qmd

```diff
@@ -197,16 +199,56 @@ hosp_model.run(
     rng_key=jax.random.key(54),
     mcmc_args=dict(progress_bar=False),
 )
+```
+
+```{python}
+# | code-fold: true
+import arviz as az
+import matplotlib.pyplot as plt
+
+
+# Retrieve the posterior samples from the model
+ppc_samples = hosp_model.posterior_predictive(
+    n_datapoints=daily_hosp_admits.size
+)
+
+# Create an InferenceData object from model
+idata = az.from_numpyro(
+    posterior=hosp_model.mcmc,
+    posterior_predictive=ppc_samples,
+)
 
-# Plotting the posterior
-out = hosp_model.plot_posterior(
-    var="latent_hospital_admissions",
-    ylab="Hospital Admissions",
-    obs_signal=daily_hosp_admits.astype(float),
+# Convert hospital admissions from discrete to continuous
+idata.posterior_predictive["negbinom_rv"] = idata.posterior_predictive[
```

Comment:
I assume this is necessary, but it's surprising to me.

Reply:
Need to check again actually. This might be an artifact from having first used `plot_ppc`, which differentiates plots based on whether the data received are integers or floats.